backend/src/services/bambora.js

```diff
@@ -22,11 +22,16 @@ module.exports = {
     },
 
     createBamboraPaymentRequest: (paymentModel) => {
+        const amount = paymentModel.amount * 100;
+        const preTaxAmount = Math.round(Number((amount * 10 / 11).toFixed(2)));
```

Comment:
Is this conversion necessary?

Reply:
If you need to transform a lot of data, you can define a function that "pipe" a value through a pipeline of transformational pure functions.

Reply:
This is a temporary price calculation logic. will be replaced later when the proper prices are available from grynos.